package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alpheios-core",
-  "version": "1.0.2",
+  "version": "1.0.2-dev",
```

Comment:
I think we have dropped this approach to versioning now. It should just be 1.0.2

Reply:
I changed it manually, I thought that it would be right.
Will change